core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -50,6 +50,7 @@
 import android.widget.ImageView;
 import android.widget.LinearLayout;
 import android.widget.Toast;
+
```

Comment:
Could you please remove these extra blank lines that have been inserted? There's no need for them.